traits/etsconfig/tests/test_etsconfig.py

```diff
@@ -8,7 +8,10 @@
 import sys
 import tempfile
 import time
-import unittest
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
The `unittest_tools` module imports stuff from `traits`, and `etsconfig` is explictly not dependent on traits so things that it provides (eg. application home directories) can be used in code which doesn't want to use the full traits libraries.

ETSConfig is in traits basically for historical reasons - it used to be part of a lower-level base library that traits depended on, but which was removed as part of the Great Renaming.